cmd/nerdctl/build.go

```diff
@@ -360,7 +363,16 @@ func generateBuildctlArgs(cmd *cobra.Command, buildkitHost string, platform, arg
 		return "", nil, false, "", nil, nil, err
 	}
 	if _, err := os.Lstat(filepath.Join(absDir, file)); err != nil {
-		return "", nil, false, "", nil, nil, err
+		if file == buildkitutil.DefaultDockerfileName {
+			logrus.Infof("%s not found, checking if Containerfile exists and overriding it", buildkitutil.DefaultDockerfileName)
+			if _, err := os.Lstat(filepath.Join(absDir, Containerfile)); err == nil {
```

Comment:
What should we do when `err != nil && !errors.Is(err, os.ErrNotExist)`

Reply:
Returned error in these cases.